go.mod

```diff
@@ -10,107 +10,322 @@ go 1.15
 
 require (
```

Comment:
the large number of changes in this file is unexpected... did you update kube-openapi with:
```
hack/pin-dependency.sh k8s.io/kube-openapi 37f25a7e63d9
hack/update-vendor.sh
```

Reply:
Yeah. It wasn't very clean though. I used go1.14.4. Maybe that was the problem?

Reply:
Something is up with update-vendor.  I dropped the version bump commit and attempting to recreate it:

``
<rebased on lastest master>
make clean
gvm use go1.15rc2
hack/pin-dependency.sh k8s.io/kube-openapi daf3cbb84823c8bd14bef3369ce4acd55ca8e9b7
```

All the go.mod changes look okay at this point.

```
hack/update-vendor.sh
```

results in a bunch of seemingly random build changes, e.g.:

```
	modified:   pkg/kubelet/cm/BUILD
	modified:   pkg/kubelet/dockershim/cm/BUILD
...

-        "@io_bazel_rules_go//go/platform:illumos": [
-            "//vendor/github.com/mrunalp/fileutils:go_default_library",
-        ],

```

Not sure why vendor is removing platform:illumos rules..

Reply:
Anyway, I was able to remove the BUILD changes manually and the vendor commit should look reasonable now.